src/afrolabs/components/kafka.clj

```diff
@@ -1250,10 +1278,14 @@
 (-prom/register-metric (prom/counter ::consumer-poll
                                      {:description "Incemented every time a poll call is completed on the consumer."
                                       :labels [:consumer-group-id]}))
+
+(-prom/register-metric (prom/summary ::consumer-lag:measure-cost
+                                     {:description "Measurement of how many milliseconds it takes to measure the consumer lag and report it."
+                                      :labels [:consumer-group-id]}))
 ;; part of commented-out code below.
 ;; This technique of determining lag on the consumer is not smart.
 ;; It should probably be a strategy.
-#_(-prom/register-metric (prom/gauge ::consumer-partition-lag
+(-prom/register-metric (prom/gauge ::consumer-partition-lag
                                    {:description "The lag in offsets per consumer-group-id and partition."
                                     :labels [:consumer-group-id
                                              :topic
```

Comment:
We are trying a way to make a metric out of consumer partition lag. This is being done inline, so I want to know the cost of the measurement itself.